zebra-rpc/src/methods.rs

```diff
@@ -810,6 +888,20 @@ pub enum GetRawTransaction {
     },
 }
 
+/// Response to a `getaddressutxos` RPC request.
+///
+/// See the notes for the [`Rpc::get_address_utxos` method].
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct GetAddressUtxos {
+    address: String,
+    txid: String,
+    height: u32,
+    #[serde(rename = "outputIndex")]
+    output_index: usize,
+    script: String,
+    satoshis: i64,
```

Comment:
Minor: Can we use the types here, rather than `String`/`i64`?

Using specific types makes it impossible to mix up the fields accidentally.

```suggestion
    address: transparent::Address,
    #[serde(with = "hex")]
    txid: transaction::Hash,
    height: u32,
    #[serde(rename = "outputIndex")]
    output_index: usize,
    #[serde(with = "hex")]
    script: transparent::Script,
    satoshis: Amount<NonNegative>,
```

Reply:
I think this is a good idea but probably will not work if we don't add more code to make it happen. We can leave it as it and open a ticket to do the change it in the future.